Makefile

```diff
@@ -445,13 +445,23 @@ ifeq ($(CI), true)
 	gocov convert $(REPORTS)/$(TEST_SCENARIO)_coverage.out | gocov-xml > $(REPORTS)/$(TEST_SCENARIO)_coverage.xml
 endif
 
-unit-test:
+docker-db:
 	docker ps -q --filter "name=postgres" | xargs -r docker kill && sleep 3
 	docker run -d  --rm --tmpfs /var/lib/postgresql/data --name postgres -e POSTGRESQL_ADMIN_PASSWORD=admin -e POSTGRESQL_MAX_CONNECTIONS=10000 -p 127.0.0.1:5432:5432 \
 	quay.io/edge-infrastructure/postgresql-12-centos7:latest
 	timeout 5m ./hack/wait_for_postgres.sh
-	SKIP_UT_DB=1 $(MAKE) _test TEST_SCENARIO=unit TIMEOUT=30m TEST="$(or $(TEST),$(shell go list ./... | grep -v subsystem))" || (docker kill postgres && /bin/false)
+
+just-unit-test:
+	SKIP_UT_DB=1 $(MAKE) _test TEST_SCENARIO=unit TIMEOUT=30m TEST="$(or $(TEST),$(shell go list ./... | grep -v subsystem))"
+
+ci-unit-test:
```

Comment:
What happens if you run `skipper make ci-unit-test` locally?
Does that work?

Reply:
yes, I have tested running it locally and it works.

Reply:
This is eventually the direction I wanted to go for both ci and local, but the concern was that people could previously run unit tests without skipper and that lead to things being changed on their local system.

Relates to https://github.com/openshift/assisted-service/pull/2916#discussion_r750471511

@omertuc does this look like what you were thinking in the last PR?

Reply:
Yeah this PR pretty much looks like what I had in mind, apart from the change of the `-build` Dockerfile base image - I expected the base image to stay the same with the `-build` Dockerfile installing postgres

Reply:
Yeah, turned out installing go in a PG image was easier than installing PG in a go image :man_shrugging: